experimental/adaptive-tool/client/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "es2019",
```

Comment:
it is better to keep the "lib" and "target" be consistent with other typescript projects.

Reply:
fixed